artifactory/commands/transferfiles/delayedartifactshandler_test.go

```diff
@@ -0,0 +1,89 @@
+package transferfiles
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/tests"
+	"github.com/jfrog/jfrog-client-go/utils/io/fileutils"
+	"github.com/stretchr/testify/assert"
+	"math"
+	"sync"
+	"testing"
+)
+
+func TestDelayedArtifactsMng(t *testing.T) {
+	// Set testing environment
+	cleanUpJfrogHome, err := tests.SetJfrogHome()
+	assert.NoError(t, err)
+	defer cleanUpJfrogHome()
+
+	artifactsNumber := 40
+	// We reduce the maximum number of entities per file to test the creation of multiple delayed artifacts files.
+	originalMaxArtifactsInFile := maxDelayedArtifactsInFile
+	maxDelayedArtifactsInFile = 20
+	defer func() { maxDelayedArtifactsInFile = originalMaxArtifactsInFile }()
```

Comment:
Following tests won't fail, but maybe if we add more tests later it can effects them? cause we change this variable, if we add integration tests in the future it can cause unexpected behavior no?